src/app/recipes/recipes-cards/recipes-cards.component.ts

```diff
@@ -3,16 +3,20 @@ import { Store } from '@ngrx/store';
 import { Observable } from 'rxjs';
 import RecipesSelectors from 'src/app/store/recipes/recipes.selector';
 import { AppState } from 'src/app/store/store';
-import Recipe from '@models/Recipe.model';
+import Recipe from '@models/recipe.model';
 
 @Component({
 	selector: 'app-recipe-cards',
 	templateUrl: 'recipes-cards.component.html',
 	styleUrls: ['recipes-cards.component.scss'],
 })
 export class RecipesCardsComponent {
-	public recipes$: Observable<Recipe[]> = this.store.select(
-		RecipesSelectors.selectRecipes
-	);
-	constructor(private store: Store<AppState>) {}
+	public recipes$: Observable<Recipe[]>;
+	public recipesAreLoaded$: Observable<boolean>;
+	constructor(private store: Store<AppState>) {
+		this.recipes$ = this.store.select(RecipesSelectors.selectRecipes);
+		this.recipesAreLoaded$ = this.store.select(
+			RecipesSelectors.selectRecipesAreLoaded
```

Comment:
move all this code from constructor to ngOnInit

https://javascript.plainenglish.io/difference-between-constructor-and-ngoninit-in-angular-537ecfa6ce1e